### PR TITLE
feature: radio field

### PR DESCRIPTION
**Pull request by thiagoyoussef**
Fixes https://github.com/avo-hq/avo/issues/3140

### CODE REVIEW

File: docs/3.0/fields/radio.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+```ruby
+{
+  admin: "Administrator",
+  manager: "Manager",
+  writer: "Writer"
+}
+```
+
+</Option>
```
Comment (Paul-Bob) [inline docs/3.0/fields/radio.md:52]: Let's mention about the options being executed within `Avo::ExecutionContext` and that options can be passed as a Proc with access to resource record etc... Like here https://docs.avohq.io/3.0/actions.html#actions-authorization

**Review by Paul-Bob — CHANGES_REQUESTED** [review]
Looking good, let's just add the `Avo::ExecutionContext` mention

File: docs/3.0/fields/radio.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  options: {
+    admin: "Administrator",
+    manager: "Manager",
+    writer: "Writer"
+  }
+```
+
+<Option name="`options`">
+
+The `options` attribute accepts either a `Hash` or a proc, allowing the incorporation of custom logic. Within this block, you gain access to all attributes of [`Avo::ExecutionContext`](../execution-context) along with the `action`, `resource`, and `view`.
```
Comment (Paul-Bob) [inline docs/3.0/fields/radio.md:28]:
```suggestion
The `options` attribute accepts either a `Hash` or a proc, allowing the incorporation of custom logic. Within this block, you gain access to all attributes of [`Avo::ExecutionContext`](../execution-context) along with the `record`, `resource`, `view` and `field`.
```

**Review by Paul-Bob — APPROVED** [review]
LGTM!

File: docs/3.0/fields/radio.md (hunk 1 above)
Comment (adrianthedev) [reply docs/3.0/fields/radio.md]: @thiagoyoussef and @Paul-Bob.

Avo follows [Rails' way of declaring options](https://apidock.com/rails/ActionView/Helpers/FormOptionsHelper/options_for_select) where the label is the key of the hash and the id is the value of the hash.
I know, I don't agree with it, but it is what it is.

Can you please check and see that we use the same way going forward on select and radio fields?

Quite thought: Maybe we switch them in Avo 4?

Comment (adrianthedev) [reply docs/3.0/fields/radio.md]: You're right. Let's leave it like that.

**Pull request merged**